applications/nrf_desktop/sample.yaml

```diff
@@ -2,36 +2,36 @@ sample:
   name: nRF52 Desktop
   description: BLE HID device reference design
 tests:
-  test:
+  applications.nrf_desktop:
     build_only: true
     build_on_all: true
     platform_whitelist: nrf52840gmouse_nrf52840 nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf52840dongle_nrf52840 nrf52kbd_nrf52832 nrf52dmouse_nrf52832 nrf52810dmouse_nrf52810
     tags: bluetooth ci_build
-  test_zdebug_b0:
+  applications.nrf_desktop.zdebug_b0:
     build_only: true
     build_on_all: true
     platform_whitelist: nrf52840gmouse_nrf52840 nrf52840dk_nrf52840 nrf52840dongle_nrf52840
     tags: bluetooth ci_build
     extra_args: "CMAKE_BUILD_TYPE=ZDebugB0"
-  test_zdebug_splitll:
+  applications.nrf_desktop_zdebug_splitll:
```

Comment:
Should be: applications.nrf_desktop.zdebug_splitll

Reply:
Could we also rename the `applications.nrf_desktop:` test to `applications.nrf_desktop.zdebug:` (if its not a problem)? This would be useful for tests.
@joerchan, should I add commit for the fixes or you will add this as part of your commit ?